src/VisualStudio/Core/Def/Watson/FaultReporter.cs

```diff
@@ -252,8 +252,9 @@ private static string GetDescription(Exception exception)
         {
         }
 
-        // If we couldn't get a stack, do this
-        return exception.Message;
+        // If we couldn't get a stack, report a generic message.
+        // The exception message is already reported in a separate cred-scanned property.
```

Comment:
the fault description field is not client side cred-scanned by VS telemetry.  We should not report the message again here, there is already an exception message property that is cred-scanned.